src/server/modes/charts/plugins/ql/yagr.ts

```diff
@@ -43,16 +43,17 @@ const applyPlaceholderSettingsToYAxis = ({
     return {scale};
 };
 
-export default ({shared}: {shared: QlConfig}) => {
-    const config = mapQlConfigToLatestVersion(shared);
+export default ({shared, ChartEditor}: {shared: QlConfig; ChartEditor: IChartEditor}) => {
+    const config = mapQlConfigToLatestVersion(shared, {i18n: ChartEditor.getTranslation});
 
     const type = (config.visualization.highchartsId || config.visualization.id) as ChartType;
 
     const percent =
         config.visualization.id === 'area100p' || config.visualization.id === 'column100p';
 
-    const tracking = (config.visualization.highchartsId ||
-        config.visualization.id) as TooltipOptions['tracking'];
+    const visualizationId = config.visualization.highchartsId || config.visualization.id;
+
+    const tracking = visualizationId === 'area' ? 'area' : 'sticky';
```

Comment:
Minor: I think we have an enum for 'area'

Reply:
I don't think this is quite right. The fact that the type from Yagr matches our visualizationId is a coincidence rather than a pattern.

My suggestion is to leave it as it is, so as not to mix up our `area` and what Yagr means by it.